spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/DeploymentPropertiesUtils.java

```diff
@@ -94,6 +99,39 @@ private DeploymentPropertiesUtils() {
 				(fromWildcard, fromApp) -> fromApp
 				)
 			);
+
+		Map<String, String> deprecated = extractDeprecatedDeployerProperties(input, appName);
+		if (deprecated.isEmpty()) {
+			return result;
+		} else {
+			deprecated.entrySet().forEach(kv -> {
+				logger.warn("Usage of application property prefix 'spring.cloud.deployer' to pass properties to the deployer has been deprecated and will be removed soon\n" +
```

Comment:
wouldn't want to be that specific in a deprecation note, as long as we warn anyone using this we can remove it in a future release